app/views/search/import_results.html.erb

```diff
@@ -0,0 +1,35 @@
+<table class="tablesorter">
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Count</th>
+      <th>Author(s)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @books.each do |b| %>
+      <tr>
+        <td>
+          <%= link_to b.title, book_path(b.id) %>
+          <%= link_to "edit", edit_book_path(b), class: "meh-btn right" %>
+          <%= link_to "delete", book_path(b), method: :delete, data: {confirm: "Are you sure you want to delete this book?"}, class: "danger-btn right" %>
+        </td>
```

Comment:
This is gonna drive librarians nuts - we need to replace with with a "remove copy" button if there is more than one copy of the book